### PR TITLE
AC Test Fix - LRAC-9658 | master

### PR DESCRIPTION
**Pull request by AfonsoHSantos**
https://issues.liferay.com/browse/LRAC-9669 (refactor)
https://testray.liferay.com/home/-/testray/runs?testrayBuildId=1575572565 (run)
afonsohsantos#47

### CODE REVIEW

**Comment by AfonsoHSantos** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/AssetsDocumentsAndMedia.testcase [hunk 1]
```diff
@@ -666,7 +666,7 @@ definition {
 
 		ACTimeFilter.setLast24Hours();
 
-		ACNavigation.openItem(itemName = "DM AC Title");
+		ACNavigation.openItem(itemName = "Document_1.jpg");
```
Comment (manuferreira) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/AssetsDocumentsAndMedia.testcase:669]: Hey @AfonsoHSantos I think this test should be on quarantine, there is an open ticket that says the document name changes in AC, see [LRAC-7890](https://issues.liferay.com/browse/LRAC-7890)

Comment (AfonsoHSantos) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/AssetsDocumentsAndMedia.testcase:669]: Hi @manuferreira , I'll move the test to quarantine, thanks!

**Comment by AfonsoHSantos** [comment]
ci:test:sf

**Pull request closed without merging**